src/server/models/app-models.js

```diff
@@ -182,6 +185,48 @@ export default function (db) {
 
   const ParticipantAllergy = db.define('participant_allergy')
 
+  const AuditClientData = db.define(
+    'audit_client_data',
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      ipVersion: {
+        // allowed values: "ivp4" and "ipv6"
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      ipAddress: {
+        type: Sequelize.BIGINT,
```

Comment:
Since we don't really need any kind of processing of the IP-address, it's only there for reference, we could just store it as a string and not worry about the ip address library and the ip versions

Reply:
Done. Storing it as binary/number makes filtering based on hostmask much easier but if we need to implement things like that at a scale we can always change it later. I changed it to be a string for now.